dev_guide/routes.adoc

```diff
@@ -133,6 +133,18 @@ TLS termination and a default certificate, which may not match the requested
 host name, resulting in validation errors.
 ====
 
+[[creating-routes-shards]]
+
+== Creating Routes with router shards
+
+In case us use the router sharding possibility 
+link: ../install_config/install/deploy_router.adoc#creating-router-shards[Creating Router Shards]
+you need to add the labels for the dedicated routes.
+
+----
+oc expose svc/low-sla-frontend --hostname=low-sla.example.com --labels='sla=low'
```

Comment:
- This command needs a `$` prompt.
- Because these docs are sometimes displayed very narrowly, we prefer to use backslash continuation style.  For example:
```bash
$ some-very-long-command \
    --interesting-option \
    --more args
```
(There are four SPC characters at the beginning of each continuation line.)